source/virtualBuffers/gecko_ia2.py

```diff
@@ -26,6 +26,9 @@ def _normalizeControlField(self,attrs):
 		ariaCurrent = attrs.get("IAccessible2::attribute_current")
 		if ariaCurrent != None:
 			attrs['current']= ariaCurrent
+		placeholder = self._getPlaceholderAttribute(attrs, "IAccessible2::attribute_placeholder")
+		if placeholder != None:

```

Comment:
Yeah I think its worth differentiating from an empty string, which would indicate an authoring or browser error. Those errors will be harder to track down if we consider None and "" as equivalent.